### DISCUSSION TITLE
Bienvenue dans les discussions Coding-UI !

### DISCUSSION THREAD

**Discussion by codingtuto** [codingtuto/Coding-UI#1] [Announcements]
## 👋 Bienvenue !
  Nous utilisons Discussions comme lieu de connexion avec d'autres membres de notre communauté. Nous espérons que vous :
  * Posez les questions que vous vous posez.
  * Partager des idées.
  * S'engager avec d'autres membres de la communauté.
  * Accueillir les autres et être ouvert d'esprit. N'oubliez pas qu'il s'agit d'une communauté que nous
  construire ensemble 💪.

  Pour commencer, commentez ci-dessous avec une présentation de vous-même et parlez-nous de ce que vous faites avec cette communauté.

**Comment by codingtuto** [comment]
**Vos suggestions de bug ou d'incompréensions sur Coding UI seront les bienvenues**

**Reply by flosspk** [reply]
Sorry but I can only communicate in English. I have used V1.4 in one of my blog and I found one little issue. It is not a huge one though. In the sidebar menu, when we hover arrow on any link, it shows the title. However, if we hover arrow key on dropdown link, it neither shows its title nor it considers it as a link unless we expand the sidebar. Only then I am able to click it as a link. When we click on drop down link when sidebar is small, then the title does appear but it is not in front of the main drop down link. Rather it is in front of home link.
